src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores-export-button.component.ts

```diff
@@ -3,68 +3,127 @@ import { roundValueSpecifiedByCourseSettings } from 'app/shared/util/utils';
 import { AlertService } from 'app/core/util/alert.service';
 import { ProgrammingExerciseStudentParticipation } from 'app/entities/participation/programming-exercise-student-participation.model';
 import { Exercise, ExerciseType, getCourseFromExercise } from 'app/entities/exercise.model';
-import { Component, Injectable, Input } from '@angular/core';
+import { Component, Injectable, Input, OnInit } from '@angular/core';
 import { ResultService } from 'app/exercises/shared/result/result.service';
 import { ProgrammingExercise } from 'app/entities/programming-exercise.model';
 import { GradingCriterion } from 'app/exercises/shared/structured-grading-criterion/grading-criterion.model';
 import { ResultWithPointsPerGradingCriterion } from 'app/entities/result-with-points-per-grading-criterion.model';
 import { faDownload } from '@fortawesome/free-solid-svg-icons';
 import { ExportToCsv } from 'export-to-csv';
+import { Feedback } from 'app/entities/feedback.model';
+
+interface TestCaseResult {
+    testName: string;
+    testResult: string;
+}
 
 @Component({
```

Comment:
This should probably get its own file, before growing even more

Reply:
Good point.. Has now been extracted into `test-case-result.model.ts` 👍

Reply:
Did you revert that change?

Reply:
@JohannesStoehr If you mean the extraction into `test-case-result.model.ts` then no.. The previous interface is now, as suggested, an individual file with the respective class..

Reply:
Ah seams like a misunderstanding. I meant extracting the html code into an own html file like it's done for almost every other component

Reply:
Oh I see... I now extracted it to an own template file  🙂